TYPO3.Neos/Classes/TYPO3/Neos/Command/SiteCommandController.php

```diff
@@ -229,11 +330,48 @@ public function listCommand()
         }
 
         $this->outputLine();
-        $this->outputLine(' ' . str_pad('Name', $longestSiteName + 15) . str_pad('Node name', $longestNodeName + 15) . 'Resources package');
-        $this->outputLine(str_repeat('-', $longestSiteName + $longestNodeName + $longestSiteResource + 15 + 15 + 2));
+        $this->outputLine(' ' . str_pad('Name', $longestSiteName + 15) . str_pad('Node name', $longestNodeName + 15) . str_pad('Resources package', $longestSiteResource + 15) . 'Status ');
+        $this->outputLine(str_repeat('-', $longestSiteName + $longestNodeName + $longestSiteResource + 7 + 15 + 15 + 15 + 2));
         foreach ($availableSites as $site) {
-            $this->outputLine(' ' . str_pad($site['name'], $longestSiteName + 15) . str_pad($site['nodeName'], $longestNodeName + 15) . $site['siteResourcesPackageKey']);
+            $this->outputLine(' ' . str_pad($site['name'], $longestSiteName + 15) . str_pad($site['nodeName'], $longestNodeName + 15) . str_pad($site['siteResourcesPackageKey'], $longestSiteResource + 15) . $site['status']);
         }
         $this->outputLine();
     }
+
+    /**
+     * Activate a site
+     *
+     * @param string $siteNodeName The node name of the site to activate
+     * @return void
+     */
+    public function activateCommand($siteNodeName)
```

Comment:
Please let's stay consistent with the command argument name here, like in `site:export`: even though `$siteNodeName` is more correct from the PHP side, we once decided that `--site-node` is better for the CLI side.

`site:prune` unfortunately defers from that rule already, but that's another story.